### PR TITLE
expect, newgrp, powertop: add Polish translation

### PR DESCRIPTION
**Pull request by rozie**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pl/linux/powertop.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+> Optymalizuje zużycie energii akumulatora.
+> Więcej informacji: <https://github.com/fenrus75/powertop>.
+
+- Kalibracja pomiarów zużycia energii
+
+`sudo powertop --calibrate`
+
+- Generowanie raportu zużycia enrgii w HTML w bieżącym katalogu:
+
+`sudo powertop --html={{power_report.html}}`
```
Comment (navarroaxel) [inline pages.pl/linux/powertop.md:12]: could `power_report` be translated?

File: pages.pl/linux/newgrp.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# newgrp
+
+> Przełącza członkostwo podstawowej grupy.
+> Więcej informacji: <https://manned.org/newgrp>.
+
+- Zmień podstawową grupę użytkownika:
+
+`newgrp {{group_name}}`
```
Comment (navarroaxel) [inline pages.pl/linux/newgrp.md:8]: could you translate `group_name` ?

Comment (rozie) [reply pages.pl/linux/newgrp.md]: Done

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**